src/states/adult.rs

```diff
@@ -416,16 +415,23 @@ impl Base for Adult {
 
         // We should only be connected to our own Elders - send to all of them
         // Need to collect IDs first so that self is not borrowed via the iterator
-        let target_ids: Vec<_> = self.peer_map.connected_ids().cloned().collect();
+        //
+        // WIP: this is probably out of date? How else do we know which our section members are?
```

Comment:
Is this still WIP?

Reply:
Not for this PR, WIP is for Part 3 and I'll try just removing the whole function as per your other comment